core/src/retransmit_stage.rs

```diff
@@ -341,6 +382,16 @@ fn retransmit(
                 None => continue,
             };
             max_slot = max_slot.max(shred_slot);
+
+            if let Some(rpc_subscriptions) = rpc_subscriptions {
+                notify_first_shred_received(
+                    shred_slot,
+                    rpc_subscriptions,
+                    sent_received_slot_notification,
+                    &root_bank,
+                );
+            }
```

Comment:
@carllin Would it be ok if we move this rpc notification to window-service? 
Or is there some reason this notification is sent in retransmit-stage?